From 214e118384d0456db19faa0fa63bff571d0229c3 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Dan=20=C4=8Cerm=C3=A1k?= <dcermak@suse.com>
Date: Mon, 3 Jan 2022 12:08:09 +0100
Subject: [PATCH 10/11] Add support for Ruby 3.1

The biggest change comming with Ruby 3.1 is that net/ftp is no longer bundled
with Ruby but distributed as a separate gem.

(cherry picked from commit a1a002831e8e9b40f8c2ba42187ace707b0cc91c)
---
 vagrant.gemspec | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/vagrant.gemspec b/vagrant.gemspec
index 2bef1036a..ccf76011a 100644
--- a/vagrant.gemspec
+++ b/vagrant.gemspec
@@ -12,7 +12,7 @@ Gem::Specification.new do |s|
   s.summary       = "Build and distribute virtualized development environments."
   s.description   = "Vagrant is a tool for building and distributing virtualized development environments."
 
-  s.required_ruby_version     = ">= 2.6", "< 3.1"
+  s.required_ruby_version     = ">= 2.6", "< 3.2"
   s.required_rubygems_version = ">= 1.3.6"
 
   s.add_dependency "bcrypt_pbkdf", "~> 1.1"
@@ -24,6 +24,7 @@ Gem::Specification.new do |s|
   s.add_dependency "listen", "~> 3.6"
   s.add_dependency "log4r", "~> 1.1.9", "< 1.1.11"
   s.add_dependency "mime-types", "~> 3.3"
+  s.add_dependency "net-ftp", "~> 0.1"
   s.add_dependency "net-ssh", ">= 6.1.0", "< 6.2"
   s.add_dependency "net-sftp", "~> 3.0"
   s.add_dependency "net-scp", "~> 3.0.0"
-- 
2.34.1

